scripts/CMakeLists.txt

```diff
@@ -52,9 +52,18 @@ if (INSTALL_SYSTEM_FILES)
 			DESTINATION ${BASH_COMPLETION_COMPLETIONSDIR}
 			RENAME kdb)
 
+	set (FISH_COMPLETION_COMPLETIONSDIR "/usr/share/fish/vendor_completions.d")
+	find_package (PkgConfig)
+	if (PKG_CONFIG_FOUND)
+		include (LibFindMacros)
+		pkg_get_variable (FISH_COMPLETION_COMPLETIONSDIR fish completionsdir)
+	endif ()
+	install (FILES kdb.fish
```

Comment:
Is this not already the case? The whole code is in inside the conditional `if (INSTALL_SYSTEM_FILES)` after all.